plugins/buildingplan.cpp

```diff
@@ -864,7 +864,7 @@ struct buildingplan_room_hook : public df::viewscreen_dwarfmodest
             && last_token <= interface_key::STRING_A058)
```

Comment:
Good catch! I think the key check here is also too lenient (it should accept keys from '1' to '9', but accepts them from '0' to ':' inclusive), so I'll change that.